openshift/patches/013-sinkbinding_webhook_corrections.patch

```diff
@@ -8,6 +8,6 @@ index eebad41b7..84afe81a6 100644
          - name: SINK_BINDING_SELECTION_MODE
 -          value: "exclusion"
 +          value: "inclusion"
- 
-         securityContext:
-           allowPrivilegeEscalation: false
+         - name: POD_NAME
```

Comment:
why do we add/remove these?  

the exclusion/inclusion I get

Reply:
hmmm, you're right it shouldn't :thinking:

Reply:
ah, it's the diff on the patch itself -- the patch diff is still s/exclusion/inclusion

Reply:
OK... :pray:  

/lgtm
/approve